src/lib/output/themes/default/partials/icon.tsx

```diff
@@ -106,36 +144,63 @@ export const icons: Record<
     },
     [ReflectionKind.Interface]: () =>
         kindIcon(
-            <path
-                d="M9.51 16V15.016H11.298V8.224H9.51V7.24H14.19V8.224H12.402V15.016H14.19V16H9.51Z"
-                fill="var(--color-text)"
-            />,
+            <text
+                fill="var(--color-icon-text)"
+                font-family="var(--icon-font-family)"
+                font-size="var(--icon-font-size)"
+                font-weight="var(--icon-font-weight)"
+                font-style="var(--icon-font-style)"
+                x="10"
+                y="18"
+            >
+                I
```

Comment:
This doesn't appear to be quite centered to me.

![image](https://github.com/user-attachments/assets/d87a0f3f-2d2c-4479-ac4d-faeed507084c)

Reply:
It appears that SVG has a way to just say "center this text"

```html
<text x="50%" y="50%" dominant-baseline="central" text-anchor="middle">I</text>
```

Reply:
> It appears that SVG has a way to just say "center this text"

There is, but the way things are constructed currently, that doesn't work. With no other changes, when I set the x/y properties to 50%, they render like this. I'm not really sure why.
<img width="129" alt="Screenshot 2024-10-10 at 2 35 09 PM" src="https://github.com/user-attachments/assets/c61c38db-d63e-4dc2-ba70-f3cbf7e377fa">

Reply:
That's because of the dominant-baseline and text-anchor properties - without them, the icons don't use the center of the letter as the rendered center